app/graph/types/feed_type.rb

```diff
@@ -5,13 +5,46 @@
   interfaces [NodeIdentification.interface]
 
   field :dbid, types.Int
+  field :user_id, types.Int
+  field :team_id, types.Int
   field :name, types.String
+  field :description, types.String
   field :published, types.Boolean
   field :filters, JsonStringType
   field :current_feed_team, FeedTeamType
   field :teams_count, types.Int
   field :requests_count, types.Int
   field :root_requests_count, types.Int
+  field :tags, types[types.String]
+  field :licenses, types[types.Int]
+  field :saved_search_id, types.Int
+
+  field :user do
+    type -> { UserType }
+
+    resolve -> (feed, _args, ctx) {
+      feed.user.then do |user|
+        ability = ctx[:ability] || Ability.new
+        user if ability.can?(:read, user)
+      end
+    }
+  end
```

Comment:
I think that in most cases there will be no permission to load a full user object... what if it we just return the user name in a string field `user_name` with no permission check?

Reply:
Based on Figma design there is no need to get user data so I removed this field